ChangeLog.md

```diff
@@ -33,6 +33,7 @@ Current Trunk
   may notice that `RelWithDebInfo` will now include debug info (it did not
   before, which appears to have been an error), and that `Release` will
   use `-O3` instead of `-O2` (which is a better choice anyhow). (#13083)
+- `SDL2_ttf` now uses upstream compiled with `TTF_USE_HARFBUZ` flag.
```

Comment:
This is in the wrong place - it should be on line 26, as it will be in the next release.

(I can't seem to add a suggestion on github to fix that, sorry. Maybe because the place it should go is outside of the diff region.)

Reply:
Oh, my bad. I thought I had moved that to the right place (I had it corrected locally). Fixed.